CONTRIBUTING.md

```diff
@@ -1,8 +1,19 @@
 # Contributing to Presto
 
-Thanks for your interest in Presto.  Our goal is to build a fast, scalable and reliable distributed SQL query engine for running low latency interactive and batch analytic queries against data sources of all sizes ranging from gigabytes to petabytes.
+Thanks for your interest in Presto. Our goal is to build a fast, scalable, and reliable distributed SQL query engine for running low latency interactive and batch analytic queries against data sources of all sizes ranging from gigabytes to petabytes.
 
-## Requirements
+# What Would You Like to Do?
+
+| Area             | Information                                                                                                                                                                                                                                                                                                                                                                        |
+|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Getting Started  | Read Presto's [open issues](https://github.com/prestodb/presto/issues). Tag issues that would make a good first pull request for new contributors with a [good first issue](https://github.com/prestodb/presto/labels/good%20first%20issue) tag. An easy way to start helping the project is to [open an issue](https://github.com/prestodb/presto/issues/new/choose). Issues can include bugs, new features, or outdated documentation.  |
```

Comment:
Can we make getting started be:

1) Build presto (remove section below, inline here).  Look for good first issue tickets.  Reference project boards for open work: https://github.com/prestodb/presto/projects?query=is%3Aopen